data/json/professions.json

```diff
@@ -1320,26 +1330,25 @@
             "PROF_POLICE"
         ],
         "items": {
-            "both": [
-                "socks",
-                "pants_army",
-                "hoodie",
-                "sneakers",
-                "gloves_tactical",
-                "glasses_bal",
-                "hat_boonie",
-                "badge_deputy",
-                "holster",
-                "wristwatch",
-                "usp_45",
-                "usp45mag",
-                "45_acp",
-                "shoulder_strap",
-                "remington_700",
-                "3006",
-                "3006",
-                "rifle_scope"
-            ],
+            "both": {
+                "items": [
+                    "socks",
+                    "pants_army",
+                    "hoodie",
+                    "sneakers",
+                    "gloves_tactical",
+                    "glasses_bal",
+                    "hat_boonie",
+                    "badge_deputy",
+                    "wristwatch"
+                ],
+                "entries": [
+                    { "item": "usp_45", "ammo-item": "45_acp", "charges": 12, "container-item": "holster" },
```

Comment:
Minor: Ideally should match map spawn (eg. use `ammo: 100`, `magazine: 100`) or something to that equivalent